drivers/gpio/relay_driver.go

```diff
@@ -23,6 +24,7 @@ func NewRelayDriver(a DigitalWriter, pin string) *RelayDriver {
 		pin:        pin,
 		connection: a,
 		high:       false,
+		inverted:   false,
```

Comment:
Seems like just making this public e.g. `Inverted` would let you get rid of the get/set functions below, since they do not do anything other than allowing the value to be changed.

What do you think?

Reply:
Yes I can do that and agree it would be better. I just tried to follow the convention used for the other fields.